modules/web/src/app/member/component.ts

```diff
@@ -233,4 +231,10 @@ export class MemberComponent implements OnInit, OnChanges, OnDestroy, AfterViewI
   isPaginatorVisible(): boolean {
     return !_.isEmpty(this.members) && this.paginator && this.members.length > this.paginator.pageSize;
   }
+
+  geturl(): string {
```

Comment:
point-1
create local variable what you have done in other places

- resourcesTypeURL -> after renaming
- etcdBackupTypeURL

use local instance variable in template

point-2
- getURLPath
- getPathEndpoint

thoughts?